src/encoded/static/components/gene.js

```diff
@@ -119,6 +119,19 @@ class Gene extends React.Component {
                           </div>
                         : null}
 
+                        {context.locations && context.locations.length > 0 ?
+                            <div data-test="locations">
+                                <dt>Gene locations</dt>
+                                <dd>
+                                    <ul>
+                                        {context.locations.map((location, l) => (
+                                            <li key={l} className="multi-comma">{`${location.assembly} ${location.chromosome}:${location.start}-${location.end}`}</li>
```

Comment:
Is the lack of space between ':' and '-' intended?

Reply:
yeah, I think so. There's a similar one in GeneticModifications for modified_site_by_coordinates so I kept the same format. Here's an example: https://www.encodeproject.org/genetic-modifications/ENCGM592MOY/